assets/js/account/b4/account-order-detail.tpl.liquid

```diff
@@ -0,0 +1,143 @@
+                            </div>
+                        </div>
+
+                    </div>
+                </div>
+                <div class="row mt-3" ng-if="$ctrl.deliveryMethod.type =='pickup'">
+                    <div class="col p-0">
+                        <div class="card">
+                            <div class="card-header text-uppercase text-condensed font-weight-bold">PICKUP INFORMATION</div>
+                            <div class="card-body">
+                                <vc-address address="$ctrl.deliveryMethod.fulfillmentCenter.address"></vc-address>
+                            </div>
+                        </div>
+
+                    </div>
+                </div>
+                <div class="row mt-3">
+                    <div class="col p-0">
+                        <div class="card">
+                            <div class="card-header text-uppercase text-condensed font-weight-bold">PAYMENT DETAILS</div>
+                            <div class="card-body">
+                                <div class="h5">
+                                    <span>{{ 'customer.order.payment_type' | t }}</span>
+                                </div>
+                                <div class="form-group mb-0">
+                                    <select name="paymentMethodSelect" id="paymentMethodSelect" ng-model="$ctrl.selectedPaymentMethodCode" ng-change="$ctrl.paymentMethodChanged()" class="form-control" ng-disabled="$ctrl.order.inPayments[0].status === 'Paid'">
+                                        <option ng-repeat="method in $ctrl.paymentMethods" ng-value="method.code" ng-bind="method.displayName"></option>
+                                    </select>
+                                </div>
+                                <div ng-if="$ctrl.order.inPayments[0].status !== 'Paid'">
+                                    <button class="btn btn-primary btn-lg" ng-click="$ctrl.payInvoice()">
+                                        <span>{{ 'customer.order.pay_invoice' | t }}</span>
+                                    </button>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="row mt-4">
+                    <div class="col-12 px-0">
+                        <hr class="mb-2 mt-0 border-1 border-gray-200" />
+                    </div>
+                    <div class="col-12 px-0" >
+                        <div>{{ 'customer.order.subscription' | t }}</div>
+                        <div>
+                            {{ 'customer.order.subscription_comment' | t }}
+                            <a ng-bind="$ctrl.order.subscriptionNumber" ui-sref="subscriptions.detail({number: $ctrl.order.subscriptionNumber})"></a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
```

Comment:
You don't need so much rows and columns (you may also use no-gutters to remove paddings from row & columns inside, but any way solution is much simpliest): just use flex for buttons and place card one by one, something like this:

![image](https://user-images.githubusercontent.com/6369252/111501805-f6561400-874d-11eb-958c-bd8e6e58e7e6.png)

The result will be exactly the same.